libpysal/graph/base.py

```diff
@@ -1484,6 +1484,42 @@ def build_travel_cost(cls, df, network, threshold, kernel=None):
         Returns
         -------
         Graph
+
+        Examples
+        ---------
+        >>> import geodatasets
+        >>> import geopandas as gpd
+        >>> import osmnx as ox
+        >>> import pandana as pdna
+        >>> from libpysal.graph import Graph
+
+        >>> # read an example geodataframe
+        >>> df = gpd.read_file(geodatasets.get_path("geoda Cincinnati")).to_crs(4326)
+
+        >>> # download a walk network using osmnx
+        >>> osm_graph = ox.graph_from_polygon(df.unary_union, network_type="walk")
```

Comment:
```suggestion
        >>> osm_graph = ox.graph_from_polygon(df.union_all(), network_type="walk")
```

unary_union is deprecated

Reply:
😬 keep forgetting that! lots of things I need to address